test/api/v3/integration/quests/POST-groups_groupid_quests_reject.test.js

```diff
@@ -180,5 +180,15 @@ describe('POST /groups/:groupId/quests/reject', () => {
       expect(rejectingMember.party.quest.key).to.not.exist;
       expect(rejectingMember.party.quest.completed).to.not.exist;
     });
+
+    it('starts the quest when the last user reject and verifies chat', async () => {
+      await leader.post(`/groups/${questingGroup._id}/quests/invite/${PET_QUEST}`);
+      await partyMembers[0].post(`/groups/${questingGroup._id}/quests/accept`);
+      await partyMembers[1].post(`/groups/${questingGroup._id}/quests/reject`);
+      await questingGroup.sync();
+
+      expect(questingGroup.chat[0].text).to.exist;
```

Comment:
All of these expects new to verify the contents of the text. So, something like:
`expect(questinGroup.chat[0].text).equals("Your quest has started....etc")`